include/gz/rendering/config.hh.in

```diff
@@ -36,11 +36,14 @@
 
 #define GZ_RENDERING_ENGINE_INSTALL_DIR "${GZ_RENDERING_ENGINE_INSTALL_DIR}"
 
-#cmakedefine HAVE_OGRE 1
-#cmakedefine HAVE_OGRE2 1
-#cmakedefine HAVE_OPTIX 1
-#cmakedefine HAVE_GAZEBO 1
-#cmakedefine INCLUDE_RTSHADER 1
+#cmakedefine GZ_RENDERING_HAVE_OGRE 1
+#cmakedefine GZ_RENDERING_HAVE_OGRE2 1
+#cmakedefine GZ_RENDERING_HAVE_OPTIX 1
 #cmakedefine GZ_RENDERING_HAVE_VULKAN 1
 
+// \todo(anyone) remove on tock
+#define HAVE_OGRE GZ_RENDERING_HAVE_OGRE
```

Comment:
To continue in the theme of protecting other folks who have their own defines, this is probably a good place to put:

```
#ifndef HAVE_OGRE
#define HAVE_OGRE GZ_RENDERING_HAVE_OGRE
#endif HAVE_OGRE

```

Reply:
ah yes added. [6881bdb](https://github.com/gazebosim/gz-rendering/pull/791/commits/6881bdbe8ebe369779cc87dac90e3c455a0d3dff)